cfme/tests/control/test_alerts.py

```diff
@@ -470,3 +470,37 @@ def test_alert_hardware_reconfigured(request, configure_fleecing, alert_collecti
         delay=30 * 60,
         additional_checks={"text": vm.name, "from_address": email}
     )
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_alert_for_diskusage():
+    """
+    Bugzillas:
+        * 1658670, 1672698
+
+    Polarion:
+        assignee: jdupuy
+        casecomponent: Control
+        caseimportance: medium
+        initialEstimate: 1/6hr
+        testSteps:
+            1. Go to Control > Explorer > Alerts
+            2. Configuration > Add new alert
+            3. Based on = Server
+            4. What to evaluate = Expression (Custom)
+            5. Driving Event = "Appliance Operation: Server High /var/www/miq/vmdb/log Disk Usage"
+            6. Assign the alert to a Alert Profile
+            7. Assign the Alert Profile to the Server
+            8. In advanced config, change:
+                  events:
+                  :disk_usage_gt_percent: 80
+                to:
+                  events:
+                  :disk_usage_gt_percent: 1
+            9. dd a file in /var/www/miq/vmdb/log large enough to trigger 1% disk usage
+        expectedResults:
+            9. the alert should fire, and the event of type
```

Comment:
**Required** You will still need steps 1-8 here, just leave those blank.

Reply:
Thanks @jawatts, I am automating this test, so I will remove this particular test_case from this PR anyway.